src/components/Dashboard/styles/PrintLayoutDashboard.module.css

```diff
@@ -1,7 +1,6 @@
-.wrapper {
+.container {
     background-color: #f4f6f8;
```

Comment:
Yet another scope-creeping comment, but should this color be fetched from `ui`? Ok, I'm sorry, I'll stop now 😉

Reply:
This color isn't in ui. I've checked with Joe on this before. Will check again if there are plans for it to go into the color suite.